### PR TITLE
Simplified CI configuration

### PR DESCRIPTION
**Pull request by ahoffer**
To test, inspect the jobs run in Circle CI for this PR.
e.g. https://circleci.com/gh/connexta/ion-search/236

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -71,13 +79,11 @@ jobs:
...
-            - Full Build
+      - Full Build:
+          context: secrets
 # TODO Find a way to suppress vulnerabilities and/or upgrade container dependencies
-#      - Scan Docker Image:
+#      - Anchore:
 #          requires:
-#            - Full Build
\ No newline at end of file
+#            - Assemble
```
Comment (jhunzik) [inline .circleci/config.yml]: ❓ This should be `Full Build` right?

Comment (ahoffer) [reply .circleci/config.yml]: Yes.

File: build.gradle [hunk 2]
```diff
@@ -90,6 +90,7 @@ configurations.all {
 dependencyCheck {
     failBuildOnCVSS = 4
     failOnError = true
+    cveValidForHours = 24
```
Comment (emmberk) [inline build.gradle]: ❓Is this setting useless for CI builds and just added to improve local builds? Or, is there something from CI build to CI build that allows this setting to work?

File: .circleci/config.yml [hunk 3]
```diff
@@ -3,52 +3,60 @@ orbs:
   anchore: anchore/anchore-engine@1.4.0
 jobs:
   Full Build:
-    machine:
+    machine: &virtual_machine
```
Comment (emmberk) [inline .circleci/config.yml]: ✏️ Is this alias needed? Same with `install_java`, `configure_java`, and `restore_cache`.

File: .circleci/config.yml [hunk 4]
```diff
@@ -3,52 +3,60 @@ orbs:
...
-          name: Building
-          command: |
-            ./gradlew clean build --info
+            sudo apt-get -qq update
+            sudo apt-get -qq install -y openjdk-11-jdk
+      - run: &configure_java
+          name: Change to Java 11
+          command: sudo update-java-alternatives --set java-1.11.0-openjdk-amd64
+      - checkout
+      - restore_cache: &restore_cache
```
Comment (emmberk) [inline .circleci/config.yml]: ✏️ Is `restore_cache` and `save_cache` needed if everything is happening in one job?

**Pull request merged**